fluentd/fluent.conf

```diff
@@ -157,7 +157,7 @@
     type record_transformer
     enable_ruby
     <record>
-      log ${(err || msg) || log}
+      log ${( (err unless (defined?(err)).nil?) || (msg unless (defined?(msg)).nil?) ) || log}
```

Comment:
Little convoluted. This would be clearer:

    (err if defined? err) || (msg if defined? msg) ) || log

Even shorter and less redundant, if more opaque:

    (err rescue nil) || (msg rescue nil) ) || log